app/views/DefaultBrowserView.tsx

```diff
@@ -48,106 +46,95 @@ const BROWSERS: IBrowsersValues[] = [
 	}
 ];
 
-interface IDefaultBrowserViewState {
-	browser: string | null;
-	supported: IBrowsersValues[];
+interface IRenderItem extends IBrowsersValues {
+	browser: string;
+	theme: TSupportedThemes;
+	changeDefaultBrowser: (newBrowser: TValue) => void;
 }
 
-type IDefaultBrowserViewProps = IBaseScreen<SettingsStackParamList, 'DefaultBrowserView'>;
+const RenderItem = memo(({ title, value, browser, changeDefaultBrowser, theme }: IRenderItem) => {
```

Comment:
This is anti pattern.
`render*` (with lowercase `r`) are functions to render stuff.
`Item` (with uppercase `I`) is a react component.
`RenderItem` is a mix of both, that's why you shouldn't do it.

It should be `Item` and on a separate file.